public/apps/configuration/panels/role-list.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React from 'react';
+import {
+  EuiPageHeader,
+  EuiTitle,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiFlexItem,
+  EuiText,
+  EuiLink,
+  EuiFlexGroup,
+  EuiButton,
+} from '@elastic/eui';
+
+export function RoleList() {
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>Roles</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentHeader id="role-table-container">
+          <EuiPageContentHeaderSection>
+            <EuiTitle>
+              <h3>Roles</h3>
+            </EuiTitle>
+            <EuiText size="s" color="secondary">
```

Comment:
why use secondary here, looks doesn't match with UI Mockup

Reply:
Updated to `subdued`, screenshot updated.